fix(README): fix Camunda properties docs

philippfromme opened the pull request:
Closes #1001

<!--

Thanks for creating this pull request!

Please make sure to link the issue you are closing as "Closes #issueNr". 
This helps us to understand the context of this PR.

-->

nikku commented on README.md:
```diff
@@ -105,14 +104,13 @@ const modeler = new BpmnModeler({
-    CamundaPlatformPropertiesProviderModule,
-    CamundaExtensionModule
+    ZeebePropertiesProviderModule,
+    camundaCloudBehaviors
```
```suggestion
    ZeebeBehaviors
```

nikku commented on README.md:
```diff
@@ -76,26 +76,25 @@ propertiesPanel.attachTo('#other-properties');
-// use Camunda BPMN namespace
-import camundaModdleDescriptors from 'camunda-bpmn-moddle/resources/camunda';
+// Camunda 8 behaviors
+import camundaCloudBehaviors from 'camunda-bpmn-js-behaviors/lib/camunda-cloud';
```
```suggestion
import ZeebeBehaviorsModule from 'camunda-bpmn-js-behaviors/lib/camunda-cloud';
```

nikku reviewed (commented):
Looks good. I added two minor suggestions (let's remove `cloud` where we can).

philippfromme commented:
> Looks good. I added two minor suggestions (let's remove `cloud` where we can).

Changed the naming. Those `cloud` and `platform` names are going to haunt us for a long time.
